src/components/Exchange/PositionsList.js

```diff
@@ -105,6 +106,7 @@ export default function PositionsList(props) {
   const [isPositionShareModalOpen, setIsPositionShareModalOpen] = useState(false);
   const [ordersToaOpen, setOrdersToaOpen] = useState(false);
   const [isHigherSlippageAllowed, setIsHigherSlippageAllowed] = useState(false);
+  const isUserAContractAddress = useIsAddressAContract();
```

Comment:
Maybe we should rename it somehow.)

Like isContractWallet / isContractAccount / isAccountAContract

Just isAddressAContract does not give any information about the address being checked

Reply:
👍